app/(dashboard)/dashboard/editor/[userId]/[eventId]/page.tsx

```diff
@@ -1,204 +1,85 @@
-                        type={eventInfo.type}
-                        name={eventInfo.label.toLowerCase()}
-                        placeholder={eventInfo.placeholder}
-                        className="my-2 w-full appearance-none rounded-md border-none"
-                      />
-                    </div>
-                  ))}
-              </div>
-              <div className="my-4 rounded-md border bg-background p-4 ">
-                {eventInfoList
-                  .filter((eventInfo) => eventInfo.type !== 'color')
-                  .map((eventInfo) => (
-                    <div key={eventInfo.id} className="mb-4">
-                      <label className="mb-2 block font-bold text-gray-700">
-                        {eventInfo.label}:
-                      </label>
-                      <div className="flex">
-                        <input
-                          type={eventInfo.type}
-                          name={eventInfo.label.toLowerCase()}
-                          placeholder={eventInfo.placeholder}
-                          className="w-full rounded-lg border px-4 py-2 focus:border-blue-300 focus:outline-none focus:ring"
-                        />
-                        {eventInfo.id > 6 && ( // Render delete button for newly added events
-                          <Button
-                            text="Delete"
-                            size="sm"
-                            className="ml-2 w-20 bg-red-500 text-white hover:bg-red-300"
-                            onClick={() => handleDeleteEventInfo(eventInfo.id)}
-                          />
-                          // <button
-                          //   type="button"
-                          //   onClick={() => handleDeleteEventInfo(eventInfo.id)}
-                          //   className="ml-2 rounded-lg bg-red-500 px-2 py-1 font-semibold text-white shadow-md hover:bg-red-600 focus:outline-none focus:ring focus:ring-red-300"
-                          // >
-                          //   Delete
-                          // </button>
-                        )}
-                      </div>
-                    </div>
-                  ))}
-              </div>
-
-              <div className="mb-4"></div>
-            </div>
-          </div>
+        <div className="p-4">
+          <WYSIWYGContainer
+            content={htmlState}
+            handleChange={handleWYSIWYGChange}
+          />
         </div>
       </div>
       <EditorRightColumn
         handleSave={handleSave}
-        handleAddEventInfo={handleAddEventInfo}
         showSavedMessage={showSavedMessage}
-        newEventLabel={newEventLabel}
-        setNewEventLabel={setNewEventLabel}
         userId={params.userId}
         eventId={params.eventId}
+        backgroundColor={backgroundColor || '#fffff'}
+        textColor={textColor || '#000000'}
+        setBackgroundColor={setBackgroundColor}
+        setTextColor={setTextColor}
       />
     </div>
   );
```

Comment:
i think i'll move all of this into a QuillEditor component. then in this page.tsx Template(params) file, call <QuillEditor params = {params}/> to add logic to move between this and the <FormEditor params = {params}/> component that i have :'0 Does that sound alright

Reply:
Would that mean that the colour selection will be in the left tab? Does the registration form not have colour selection also? We can use the same backgroundcolour and textcolour.

Reply:
Oh, my form editor color selection is still in the right bar, if we're using the same bg color and text color for both form and event then its all fine
But the handleSave of the two sections are different so i think we'll need to settle on that if we want to reuse the whole right bar